modules/swagger-parser/src/test/java/io/swagger/parser/OpenAPIParserTest.java

```diff
@@ -67,7 +67,6 @@ public void testSimple() {
 
     @Test
     public void testDuplicationOfTags() {
```

Comment:
If you implement the messages map as mentioned before, please add the validation of the message here.